requirements/build.txt

```diff
@@ -2,3 +2,4 @@ cython
 numpy
 packaging
 setuptools
+pyyaml
```

Comment:
could remove it since it is added in https://github.com/open-mmlab/mmdeploy/pull/1018/files